dev-requirements.tx

```diff
@@ -0,0 +1,3 @@
+black>=22.1.0,<22.2.0
```

Comment:
This file has a `.tx` ending, should it be `.txt`?

Reply:
thanks!